Make check target is failing in Jenkins pipeline

`make check` target is failing on the `make update` target

```
[2020-02-12T09:21:34.286Z] /var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/journalbeat/build/python-env/bin/pip: line 3: import: command not found
[2020-02-12T09:21:34.286Z] /var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/journalbeat/build/python-env/bin/pip: line 4: import: command not found
[2020-02-12T09:21:34.286Z] from: can't read /var/mail/pip._internal.cli.main
[2020-02-12T09:21:34.286Z] /var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/journalbeat/build/python-env/bin/pip: line 9: syntax error near unexpected token `('
[2020-02-12T09:21:34.286Z] /var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/journalbeat/build/python-env/bin/pip: line 9: `    sys.argv[0] = re.sub(r"(-script.pyw?|.exe)?$", "", sys.argv[0])'
[2020-02-12T09:21:34.286Z] /var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/journalbeat/build/python-env/bin/pip: line 3: import: command not found
[2020-02-12T09:21:34.286Z] /var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/journalbeat/build/python-env/bin/pip: line 4: import: command not found
[2020-02-12T09:21:34.286Z] from: can't read /var/mail/pip._internal.cli.main
[2020-02-12T09:21:34.286Z] /var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/journalbeat/build/python-env/bin/pip: line 9: syntax error near unexpected token `('
[2020-02-12T09:21:34.286Z] /var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/journalbeat/build/python-env/bin/pip: line 9: `    sys.argv[0] = re.sub(r"(-script.pyw?|.exe)?$", "", sys.argv[0])'
[2020-02-12T09:21:34.286Z] make[2]: *** [python-env] Error 2
[2020-02-12T09:21:34.286Z] Error: running "make update" failed with exit code 2
[2020-02-12T09:21:34.286Z] ../libbeat/scripts/Makefile:141: recipe for target 'check' failed
[2020-02-12T09:21:34.286Z] make[1]: *** [check] Error 2
[2020-02-12T09:21:34.286Z] make[1]: Leaving directory '/var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/journalbeat'
[2020-02-12T09:21:34.286Z] Makefile:90: recipe for target 'check' failed
```

It happens 5 days ago on a Linux box  https://beats-ci.elastic.co/blue/organizations/jenkins/Beats%2Fbeats-beats-mbp/detail/PR-15548/13/pipeline/

locally I have similar error but not the same

```
created virtual environment in 1875ms CPython2Posix(dest=/..../src/beats/build/python-env, clear=False, global=False) with seeder FromAppData pip=latest setuptools=latest wheel=latest app_data_dir=/Users/.../Library/Application Support/virtualenv/seed-v1 via=copy
Traceback (most recent call last):
  File "/Users/.../src/beats/build/python-env/bin/pip", line 5, in <module>
    from pip._internal.cli.main import main
ImportError: No module named main
make: *** [python-env] Error 1
```

Today we have a different error in the CI

```
[2020-02-17T14:00:15.756Z] 2020/02/17 14:00:15 Found Elastic Beats dir at /var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats
[2020-02-17T14:00:16.329Z] Generated fields.yml for filebeat to /var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/filebeat/build/fields/fields.all.yml
[2020-02-17T14:00:18.878Z] Traceback (most recent call last):
[2020-02-17T14:00:18.878Z]   File "/var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/filebeat/scripts/docs_collector.py", line 91, in <module>
[2020-02-17T14:00:18.878Z]     collect(beat_name)
[2020-02-17T14:00:18.878Z]   File "/var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/filebeat/scripts/docs_collector.py", line 42, in collect
[2020-02-17T14:00:18.878Z]     module_file += f.read()
[2020-02-17T14:00:18.878Z]   File "/usr/lib/python3.5/encodings/ascii.py", line 26, in decode
[2020-02-17T14:00:18.878Z]     return codecs.ascii_decode(input, self.errors)[0]
[2020-02-17T14:00:18.878Z] UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 516: ordinal not in range(128)
[2020-02-17T14:00:18.878Z] Error: running "/var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/build/ve/linux/bin/python /var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/filebeat/scripts/docs_collector.py --beat filebeat" failed with exit code 1
[2020-02-17T14:00:18.879Z] ../libbeat/scripts/Makefile:143: recipe for target 'check' failed
[2020-02-17T14:00:18.879Z] make[1]: *** [check] Error 1
[2020-02-17T14:00:18.879Z] make[1]: Leaving directory '/var/lib/jenkins/workspace/Beats_beats-beats-mbp_PR-15548/src/github.com/elastic/beats/filebeat'
[2020-02-17T14:00:18.879Z] Makefile:92: recipe for target 'check' failed
[2020-02-17T14:00:18.879Z] make: *** [check] Error 1
```

--- a/dev-tools/generate_notice.py
+++ b/dev-tools/generate_notice.py
@@ -15,13 +15,8 @@ def read_file(filename):
         print("File not found {}".format(filename))
         return ""
 
-    try:
-        with open(filename, 'r') as f:
-            return f.read()
-    except UnicodeDecodeError:
-        # try latin-1
-        with open(filename, 'r', encoding="ISO-8859-1") as f:
-            return f.read()
+    with open(filename, 'r', encoding='utf_8') as f:
+        return f.read()
 
 
 def get_library_path(license):
@@ -37,7 +32,7 @@ def get_library_path(license):
 
 def read_versions(vendor):
     libs = []
-    with open(os.path.join(vendor, "vendor.json")) as f:
+    with open(os.path.join(vendor, "vendor.json"), encoding='utf_8') as f:
         govendor = json.load(f)
         for package in govendor["package"]:
             libs.append(package)
@@ -222,11 +217,11 @@ def get_url(repo):
 def create_notice(filename, beat, copyright, vendor_dirs, csvfile, overrides=None):
     dependencies = gather_dependencies(vendor_dirs, overrides=overrides)
     if not csvfile:
-        with open(filename, "w+") as f:
+        with open(filename, "w+", encoding='utf_8') as f:
             write_notice_file(f, beat, copyright, dependencies)
             print("Available at {}".format(filename))
     else:
-        with open(csvfile, "wb") as f:
+        with open(csvfile, "wb", encoding='utf_8') as f:
             csvwriter = csv.writer(f)
             write_csv_file(csvwriter, dependencies)
             print("Available at {}".format(csvfile))

--- a/filebeat/scripts/docs_collector.py
+++ b/filebeat/scripts/docs_collector.py
@@ -38,13 +38,13 @@ This file is generated! See scripts/docs_collector.py
         module_file = generated_note
         module_file += "[[filebeat-module-" + module + "]]\n"
 
-        with open(module_doc) as f:
+        with open(module_doc, encoding='utf_8') as f:
             module_file += f.read()
 
         beat_path = os.path.join(module_dir, "_meta")
 
         # Load title from fields.yml
-        with open(beat_path + "/fields.yml") as f:
+        with open(beat_path + "/fields.yml", encoding='utf_8') as f:
             fields = yaml.load(f.read())
             title = fields[0]["title"]
 
@@ -61,8 +61,8 @@ For a description of each field in the module, see the
 """
 
         # Write module docs
-        with open(os.path.abspath("docs") + "/modules/" +
-                  module + ".asciidoc", 'w') as f:
+        docs_path = os.path.join(os.path.abspath("docs"), "modules",  module + ".asciidoc")
+        with open(docs_path, 'w', encoding='utf_8') as f:
             f.write(module_file)
 
     module_list_output = generated_note
@@ -76,7 +76,7 @@ For a description of each field in the module, see the
         module_list_output += "include::modules/" + m + ".asciidoc[]\n"
 
     # Write module link list
-    with open(os.path.abspath("docs") + "/modules_list.asciidoc", 'w') as f:
+    with open(os.path.abspath("docs") + "/modules_list.asciidoc", 'w', encoding='utf_8') as f:
         f.write(module_list_output)
 
 

--- a/libbeat/scripts/unpack_dashboards.py
+++ b/libbeat/scripts/unpack_dashboards.py
@@ -27,7 +27,7 @@ def transform_data(data, method):
 
 
 def transform_file(path, method):
-    with open(path) as f:
+    with open(path, encoding='utf_8') as f:
         data = json.load(f)
 
     transform_data(data, method)
@@ -52,5 +52,5 @@ if __name__ == "__main__":
         data = transform_file(path, method)
         new_data = json.dumps(data, sort_keys=True, indent=4)
 
-        with open(path, 'w') as f:
+        with open(path, 'w', encoding='utf_8') as f:
             f.write(new_data)

